src/pods/toolbar/components/toolbar-button/toolbarButton.component.tsx

```diff
@@ -1,23 +1,55 @@
 import React from 'react';
+import useShortcut from '../../shortcut/shortcut.hook';
+import { isMacOS } from '@/common/helpers/platform.helpers';
+
+export interface ShortcutOptions {
+  id: string;
+  targetKey: string[];
+  targetKeyLabel: string;
+  description: string;
+}
```

Comment:
i think that its more convenient move it to shortcut.model or somethink like that